app/src/main/java/ru/mikhailskiy/intensiv/ui/movie_details/MovieDetailsFragment.kt

```
@@ -5,41 +5,84 @@ import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import com.squareup.picasso.Picasso
+import com.xwray.groupie.GroupAdapter
+import com.xwray.groupie.kotlinandroidextensions.GroupieViewHolder
+import kotlinx.android.synthetic.main.movie_details_fragment.*
 import ru.mikhailskiy.intensiv.R
-
-private const val ARG_PARAM1 = "param1"
-private const val ARG_PARAM2 = "param2"
+import ru.mikhailskiy.intensiv.common.HorizontalSpaceDecoration
+import ru.mikhailskiy.intensiv.data.actor.Actor
+import ru.mikhailskiy.intensiv.data.actor.ActorMockRepository
+import ru.mikhailskiy.intensiv.data.movie.MovieDetail
+import ru.mikhailskiy.intensiv.data.movie.MovieDetailMockRepository
 
 class MovieDetailsFragment : Fragment() {
 
     private var param1: String? = null
     private var param2: String? = null
 
+    private val actorAdapter:GroupAdapter<GroupieViewHolder> by lazy {
+        GroupAdapter<GroupieViewHolder>()
+    }
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        arguments?.let {
-            param1 = it.getString(ARG_PARAM1)
-            param2 = it.getString(ARG_PARAM2)
-        }
     }
 
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
         // Inflate the layout for this fragment
+
         return inflater.inflate(R.layout.movie_details_fragment, container, false)
     }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        list_actors?.addItemDecoration(HorizontalSpaceDecoration(32))
+
+        val movieDetail = MovieDetailMockRepository.getDetail()
+        movieDetailToView(movieDetail)
+    }
+
+    private fun movieDetailToView(movieDetail: MovieDetail)
+    {
+        text_title?.text = movieDetail.title
```

Эти проверки тут лишние так как синтетики генерирует эти поля - и они not null

---

Да, могут быть ситуации когда не тот файл заинфлэйтился - тогда будет краш - но так гораздо проще будет заметить ошибку чем если поле вообще не отобразится. Тут используется принцип Fail Fast https://en.wikipedia.org/wiki/Fail-fast